.github/workflows/release.yaml

```diff
@@ -34,14 +34,15 @@ jobs:
           fetch-depth: 0
           repository: opentargets/gentropy
           token: ${{ secrets.GITHUB_TOKEN }}
+          persist-credentials: false
```

Comment:
Is this added for security reasons or because it is needed?

Reply:
https://github.com/actions/checkout/issues/485 - I have added it not to persist this token, as it's not the default one.